net-p2p/kubo/Manifest

```diff
@@ -1,4 +1 @@
-DIST kubo-0.20.0.tar.gz 15476743 BLAKE2B 8b7e58f18c1342981f575545c6e11a167df240bb0d6d0712bcb3fd6a0aa7f84a6737af06d2ae89fe998ee715be5c0e0c39f4ce83685ae238008c46f48c6cf7a1 SHA512 5231efce18dbb6491fa6665db8d1522efcb2d7278203259714b77d655060765922e9076addb1de3e10950c2c7aa3c9e24e2114879c3e4ce1fceef69755b12eaf
-DIST kubo-0.21.0.tar.gz 15523789 BLAKE2B 3db5e59ade1b121a7403b4188e90305dc1689922494114a7729ae23d3e3c8d89c244dc8ebb5a2563c79c6f591b4840d79c935cf6fbb7004e1f869c165308eeca SHA512 16b1462f4f2811f6188d6e6ca9c466bb8071dd45ca66d40e3f4c7d37d31b16bb3fb6d2408c61dfc5d3120847c25da735db0f89661d29493684a81a6f94251278
-DIST kubo-0.22.0.tar.gz 15681418 BLAKE2B 059c6b34a8188ad2add2ecb1cd8f69d755388d9c74c58ad5c2e95459101aa77397ad16e92b72ffe703f0dc2f62b04573fdf9da065aaee003623c6038601370c7 SHA512 08fff4a53286e106ecebeb2bee6f0f0154e8f75029f9dacf5c5e99f23d27c8243adb8cf69bea7da1df0e849bf325f18516ed6c18a5454623da2530a0f3810a32
```

Comment:
This would drop the current stable version, I'll merge without this and request stabilization 0.22.

Reply:
Stable version depends on a version of Go compiler no longer in the tree, 0.22 also depends on a version that is not on the tree. That's why they were removed.

Reply:
I wonder, should we just destabilize it? Or stabilize something newer faster?

Reply:
I would prefer to stabilize version 0.25.0 instead of having to backport fixes to previous versions. I'm unaware how critical it is to stabilize a new version as fast as possible. As far as I know ipfs/kubo is still evolving fast so a stable version, at first glance, doesn't seem as useful as it would be for other components. I'm interested in feedback in this regard (nowadays I use ipfs sparingly and for nothing critical).